lms/models/application_instance.py

```diff
@@ -198,3 +198,10 @@ def product(self):
             product = self.Product.UNKNOWN
 
         return product
+
+    @property
+    def lti_version(self):
```

Comment:
I'd probably add a docstring here. Without any extra context, i'd expect this to return "1.1.0" rather than "LTI-1p0". So it's probably worth mentioning why this is the way it is.